### DISCUSSION TITLE
Application not found

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#25788] [Q&A General]
Below is the output I get when I try to test my moose. I am new to moose and I was watching one of your moose workshops which led me to test and I got this out come.

[emmanuel324@tinkercliffs1 ~]$ cd ~emmanuel324/projects/moose
[emmanuel324@tinkercliffs1 moose]$ cd tutorials/darcy_thermo_mech
[emmanuel324@tinkercliffs1 darcy_thermo_mech]$ cd step01_diffusion
[emmanuel324@tinkercliffs1 step01_diffusion]$ ./run_tests
problems.step: Working Directory: /home/emmanuel324/projects/moose/tutorials/darcy_thermo_mech/step01_diffusion/problems
problems.step: Running command: 
problems.step: 
problems.step: ################################################################################
problems.step: Tester failed, reason: Application not found
problems.step: 
problems.step ................................................................. FAILED (Application not found)
kernels/simple_diffusion.test: Working Directory: /home/emmanuel324/projects/moose/tutorials/darcy_thermo_mech/step01_diffusion/tests/kernels/simple_dif
fusion
kernels/simple_diffusion.test: Running command: 
kernels/simple_diffusion.test: 
kernels/simple_diffusion.test: ################################################################################
kernels/simple_diffusion.test: Tester failed, reason: Application not found
kernels/simple_diffusion.test: 
kernels/simple_diffusion.test ................................................. FAILED (Application not found)
Final Test Results:
--------------------------------------------------------------------------------------------------------------
problems.step ................................................................. FAILED (Application not found)
kernels/simple_diffusion.test ................................................. FAILED (Application not found)
--------------------------------------------------------------------------------------------------------------
Ran 2 tests in 0.1 seconds. Average test time 0.0 seconds, maximum test time 0.0 seconds.
0 passed, 0 skipped, 0 pending, 2 FAILED
[emmanuel324@tinkercliffs1 step01_diffusion]$

**Comment by GiudGiud** [comment]
Hello

The executable seemingly did not build. Can you go in the folder with the makefile and type make there

**Reply by Emmanuel-324** [reply]
Below is the make file for the example I am trying to run. Where should I type the make?
```
###############################################################################
################### MOOSE Application Standard Makefile #######################
###############################################################################
#
# Optional Environment variables
# MOOSE_DIR        - Root directory of the MOOSE project
#
###############################################################################
# Use the MOOSE submodule if it exists and MOOSE_DIR is not set
MOOSE_SUBMODULE    := $(CURDIR)/moose
ifneq ($(wildcard $(MOOSE_SUBMODULE)/framework/Makefile),)
  MOOSE_DIR        ?= $(MOOSE_SUBMODULE)
else
  MOOSE_DIR        ?= $(shell dirname `pwd`)/../../
endif

# framework
FRAMEWORK_DIR      := $(MOOSE_DIR)/framework
ADDITIONAL_CPPFLAGS += -Wall -Wextra
include $(FRAMEWORK_DIR)/build.mk
include $(FRAMEWORK_DIR)/moose.mk

################################## MODULES ####################################
ALL_MODULES               := no

CHEMICAL_REACTIONS        := no
CONTACT                   := no
FLUID_PROPERTIES          := no
# heat_conduction is required by Steps 5+
HEAT_TRANSFER             := yes
LINEAR_ELASTICITY         := no
MISC                      := no
NAVIER_STOKES             := no
# phase_field is required by Step 10.
PHASE_FIELD               := yes
POROUS_FLOW               := no
RICHARDS                  := no
# tensor_mechanics is required by Steps 9-10.
TENSOR_MECHANICS          := yes
XFEM                      := no
include $(MOOSE_DIR)/modules/modules.mk
###############################################################################

# dep apps
APPLICATION_DIR    := $(CURDIR)
APPLICATION_NAME   := darcy_thermo_mech
BUILD_EXEC         := yes
GEN_REVISION       := no
include            $(FRAMEWORK_DIR)/app.mk

###############################################################################
# Additional special case targets should be added here
```

**Reply by GiudGiud** [reply]
in that folder where the Makefile is, just type `make` in the command line
